CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs

```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
             var daysToFriday = (7 + (DayOfWeek.Friday - currentDateTime.DayOfWeek)) % 7;
             return currentDateTime.AddDays(daysToFriday);
         }
+
+        private static string TrimEmail(string email)
```

Comment:
```suggestion
        public static string removeEmailDomainName(string email)
        {
            return email.Replace(".co.uk", "")
                .Replace(".com","");
        }
```
Would it be safer to check for madetech email without domain name?

Reply:
Could we replace `.co.uk` with `.com` or is removing both `.com` and `.co.uk` better?

Thoughts?

Reply:
I guess it's the same, either could work

Reply:
Changed the function here https://github.com/chubberlisk/crypto_tech/pull/34/commits/63fbc8542ee2c52ae40c0ae424431ce25fc30921.

Cheers

Reply:
Marking this as resolved for now. 
Please review the commit https://github.com/chubberlisk/crypto_tech/pull/34/commits/63fbc8542ee2c52ae40c0ae424431ce25fc30921 if you have any suggestions for changes.